src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -648,24 +843,29 @@ def __save_log(logs_dir, dir, test_name):
                                 ref_k = None
                                 test_cnt_real_saved_now += 1
                                 test_name = None
-                                test_log = list()
+                                test_log = []
                 log_file.close()
-                if test_name != None:
+                if test_name is not None:
                     dir = INTERAPTED_DIR
                     if __save_log(logs_dir, dir, test_name):
                         interapted_tests.add(test_name)
 
-                if (self._split_unit == constants.SUITE_UNIT_NAME):
+                if self._split_unit == constants.SUITE_UNIT_NAME:
                     test_cnt_real = len(test_suites)
                 else:
                     test_cnt_real = test_cnt_real_saved_now
 
                 if test_cnt_real < test_cnt_expected:
-                    logger.error(f"Number of {self._split_unit}s in {log}: {test_cnt_real}. Expected is {test_cnt_expected} {self._split_unit}")
+                    logger.error(
+                        f"Number of {self._split_unit}s in {log}: {test_cnt_real}. Expected is {test_cnt_expected} {self._split_unit}"
+                    )
                 else:
                     os.remove(log_filename)
 
-        if len(list(Path(os.path.join(self._working_dir, "temp")).rglob("log_*.log"))) == 0:
+        if (
+            len(list(Path(os.path.join(self._working_dir, "temp")).rglob("log_*.log")))
```

Comment:
Hm, to move to one line? Strange separation of one condition...

Reply:
fixed